torchTextClassifiers/torchTextClassifiers.py

```diff
@@ -326,184 +292,258 @@ def train(
             ),
             EarlyStopping(
                 monitor="val_loss",
-                patience=patience_train,
+                patience=training_config.patience_early_stopping,
                 mode="min",
             ),
             LearningRateMonitor(logging_interval="step"),
         ]
-        
-        train_params = {
+
+        trainer_params = {
             "callbacks": callbacks,
-            "max_epochs": num_epochs,
+            "max_epochs": training_config.batch_size,

```

Comment:
I think it chould be num_epochs instead : 
```suggestion
            "max_epochs": training_config.num_epochs,
```